src/domain/enrolment/enrolmentAuthenticationNotification/__tests__/EnrolmentAuthenticationNotification.test.tsx

```diff
@@ -0,0 +1,111 @@
+import { MockedResponse } from '@apollo/client/testing';
+import { clear } from 'jest-date-mock';
+import React from 'react';
+
+import { TEST_USER_ID } from '../../../../constants';
+import { UserDocument } from '../../../../generated/graphql';
+import { fakeUser } from '../../../../utils/mockDataUtils';
+import { fakeAuthenticatedStoreState } from '../../../../utils/mockStoreUtils';
+import {
+  configure,
+  CustomRenderOptions,
+  getMockReduxStore,
+  render,
+  screen,
+  userEvent,
+  waitFor,
+} from '../../../../utils/testUtils';
+import { hiddenStyles } from '../../../app/authenticationNotification/AuthenticationNotification';
+import userManager from '../../../auth/userManager';
+import { mockedEventResponse } from '../../../event/__mocks__/event';
+import { TEST_PUBLISHER_ID } from '../../../organization/constants';
+import { registration } from '../../../registration/__mocks__/registration';
+import { ENROLMENT_ACTIONS } from '../../constants';
+import EnrolmentAuthenticationNotification from '../EnrolmentAuthenticationNotification';
+
+configure({ defaultHidden: true });
+beforeEach(() => clear());
+
+const userVariables = {
+  createPath: undefined,
+  id: TEST_USER_ID,
+};
+
+const renderComponent = (renderOptions?: CustomRenderOptions) =>
+  render(
+    <EnrolmentAuthenticationNotification
+      action={ENROLMENT_ACTIONS.UPDATE}
+      registration={registration}
+    />,
+    renderOptions
+  );
+
+test('should show sign in notification is user is not signed in', () => {
+  renderComponent();
+});
```

Comment:
should this test also have something else?

Reply:
There was also some other empty unit tests. Removed all of them